ImprimirAutor.php

```diff
@@ -0,0 +1,9 @@
+<?php
```

Comment:
```suggestion
<?php

```
Agregar línea en blanco